--- rc_test2/site/src/routes.js	2018-09-27 13:24:59.420948623 -0500
+++ rc_test/site/src/routes.js	2018-09-27 10:11:41.466695851 -0500
@@ -182,12 +182,34 @@
   loading: Loading,
 });
 
+const DatatablesDescription = Loadable({
+  loader: () => import('./views/Datatables/DatatablesDescription'),
+  loading: Loading,
+});
+
+const DatatablesBasic = Loadable({
+  loader: () => import('./views/Datatables/DatatablesBasic'),
+  loading: Loading,
+});
 
+const DatatablesAjaxMssql = Loadable({
+  loader: () => import('./views/Datatables/DatatablesAjaxMssql'),
+  loading: Loading,
+});
+
+const DatatablesAjaxOracle = Loadable({
+  loader: () => import('./views/Datatables/DatatablesAjaxOracle'),
+  loading: Loading,
+});
 
 // https://github.com/ReactTraining/react-router/tree/master/packages/react-router-config
 const routes = [
   { path: '/', exact: true, name: 'Home', component: DefaultLayout },
   { path: '/dashboard', name: 'Dashboard', component: Dashboard },
+  { path: '/datatables/description', name: 'Datatables Description', component: DatatablesDescription },
+  { path: '/datatables/basic', name: 'Basic Datatables', component: DatatablesBasic },
+  { path: '/datatables/ajax-mssql', name: 'Ajax Datatables (MSSQL)', component: DatatablesAjaxMssql },
+  { path: '/datatables/ajax-oracle', name: 'Ajax Datatables (Oracle)', component: DatatablesAjaxOracle },
   { path: '/theme', exact: true, name: 'Theme', component: Colors },
   { path: '/theme/colors', name: 'Colors', component: Colors },
   { path: '/theme/typography', name: 'Typography', component: Typography },
